src/openforms/submissions/logic/actions.py

```diff
@@ -308,6 +309,57 @@ def get_action_log_data(
         }
 
 
+class DMNConfig(TypedDict):
+    plugin_id: str
+    input_mapping: dict[str, str]
+    output_mapping: dict[str, str]
+    definition_id: str
+    definition_version: str
+
+
+@dataclass
+class EvaluateDMNAction(ActionOperation):
+    input_mapping: dict[str, str]
+    output_mapping: dict[str, str]
+    definition_id: str
+    plugin_id: str
+    definition_version: str = ""
+
+    @classmethod
+    def from_action(cls, action: ActionDict) -> "ActionOperation":
+        dmn_config: DMNConfig = action["action"]["value"]
```

Comment:
does it make sense to use a `value` key here? I'd rather expect something like `"config"` as key or even no nesting at all. Perhaps this is something for the future since it deals with the polymorphism of logic actions :thinking:

Reply:
Initially I had done it with a 'config' key, but then saw that the `ServiceFetchAction` had the config in the value, so I followed that in order to be consistent.

Reply:
`ServiceFetchAction` is weird because it persists the configuration on the variable itself (which makes sense!) and only stores a reference to the service fetch configuration. I missed during review that it was stored in `value` too which I don't think is the best fit, there a `config` key would also be a better name.

`value` was really intended for the expression (JsonLogic) to produce the value based on other inputs - or a fixed value. It's being abused now when we added more features :grimacing:

Reply:
Ok! Then I will move it to a config key!